app/src/main/java/com/asfoundation/wallet/util/KS.java

```diff
@@ -42,7 +42,7 @@
 @TargetApi(23) public class KS {
   private static final String TAG = "KS";
 
-  private static final String ANDROID_KEY_STORE = "AndroidKeyStore";
+  public static final String ANDROID_KEY_STORE = "AndroidKeyStore";
```

Comment:
Why don't pass this to kotlin?

Reply:
It has too much sensitive code and should not be done lightly. Will consider that update when we have more time to do it.